pkg/secrets/plugins/host/plugin.go

```diff
@@ -22,6 +24,10 @@ func (p Plugin) Close() error {
 	return nil
 }
 
+func (p Plugin) Create(keyName string, keyValue string, value string) error {
+	return errors.New("not implemented")
```

Comment:
Are there any code flows that would hit this error? I'm concerned that this error message if we did accidentally hit it would not be helpful to the user. Perhaps we can say "The secrets.porter.host plugin does not support saving secrets. Please configure a secret plugin."

Reply:
I have updated the error message. Let me know if it helps!